src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,31 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
+        for j in range(cur_index+1, len(arr)):
+            if arr[cur_index] > arr[j]:
+                cur_index = j
+        arr[smallest_index], arr[cur_index] = arr[cur_index], arr[smallest_index]
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
+        # (hint, can do in 3 loc)
 
         # TO-DO: swap
 
-
-
-
     return arr
 
 
 # TO-DO:  implement the Bubble Sort function below
-def bubble_sort( arr ):
-
+def bubble_sort(arr):
+    for i in range(len(arr)-1, 0, -1):
+        for j in range(i):
+            if arr[j] > arr[j+1]:
+                arr[j], arr[j+1] = arr[j+1], arr[j]
     return arr
```

Comment:
Great work with the implementation here. Say we could only use a single loop here. How would we refactor this?